lib/auxiliary/id_gen.rb

```diff
@@ -16,9 +16,12 @@ def initialize(range, timeout)
 
     def load_data
       ids = Array.new(@range.last + 1 - @range.first, true)
-      @conn.execute 'CREATE TABLE IF NOT EXISTS ActiveIds('\
-                  'IdNumber INT PRIMARY KEY NOT NULL, PID INT NOT NULL,'\
-                  'Time TIMESTAMP NOT NULL)'
+      @conn.execute <<~SQL
+          CREATE TABLE IF NOT EXISTS ActiveIds(
```

Comment:
This has four-column indent while everywhere else has two-column indent. (So RuboCop also misses indents in sluggish heredoc? :confounded:)

Reply:
What? Then why not in `kill_if_alive`? :confused:

Reply:
Everything ok if `-SQL`, but when `~SQL` RuboCop generate offence:
```
lib/auxiliary/id_gen.rb:20:1: C: [Correctable] Layout/HeredocIndentation: Use 2 spaces for indentation in a heredoc
```

Reply:
I think I get the problem now. The closing brace is indented with 2 spaces so RuboCop does no longer complain, but the first line still has a wrong indent. It should be fixed so that it will have 2 spaces.

Reply:
Thanks for explanation, fixed